zenpy/lib/api.py

```diff
@@ -1854,3 +1861,27 @@ def responses_incremental(self, start_time):
         :param start_time: time to retrieve events from.
         """
         return self._query_zendesk(self.endpoint.responses_incremental, 'responses', start_time=start_time)
+
+class TalkApi(Api):
```

Comment:
This needs another level of nesting so it follows the same pattern as the rest of the wrapper. There is an example of how this looks in the HelpCentreAPI - https://github.com/facetoe/zenpy/blob/master/zenpy/lib/api.py#L1819. 

Also this will allow it to be extended to add support for create/updated easily enough.

Reply:
Aaaaaah ok, makes total sense, So I will have TalkAPIbase, then TalkAPI that inherits from this that has all of my endpoints which will point to things like... StatsAPI, GreetingsAPI, blah blah blah... whatever top level item that endpoint is a part of. Ok that's great.

That being said, I have one question...
self._response_handlers = (MissingTranslationHandler,) + self._response_handlers
What is MissingTranslationHandler? I don't have time to look right now but I will look on the weekend if you don't have time to respond. I am guessing it is just HelpDesk related perhaps and I just use the normal response handlers that already seem to be doing their job just fine for the talk API?

Thanks so much for the help!

Reply:
Yep, that is a HelpDesk specific thing, the default response handlers should work fine for your situation.